edk2toollib/utility_functions.py

```diff
@@ -177,9 +177,11 @@ def RunCmd(cmd, parameters, capture=True, workingdir=None, outfile=None, outstre
     endtime = datetime.datetime.now()
     delta = endtime - starttime
     endtime_str = "{0[0]:02}:{0[1]:02}".format(divmod(delta.seconds, 60))
+    returncode_str = "{0:#010x}".format(c.returncode)
     logging.log(logging_level, "------------------------------------------------")
     logging.log(logging_level, "--------------Cmd Output Finished---------------")
     logging.log(logging_level, "--------- Running Time (mm:ss): " + endtime_str + " ----------")
+    logging.log(logging_level, "----------- Return Code: " + returncode_str + " ------------")
```

Comment:
Line 188 - the other place returncode is printed should be updated to match...or maybe not even needed.

Reply:
Thanks for the feedback.  @spbrogan, I've removed the other locations where returncode is printed.
For line 188, where we raise an exception, do you have a preference of whether I leave the exception string empty or change the string?  For the time being, I've changed the string.  But I'm open to making this whatever you'd like it to be.

Reply:
looks fine for now.  I don't have a strong opinion.

Reply:
Thanks!